src/test/java/seedu/duke/data/EditCommandTest.java

```diff
@@ -0,0 +1,64 @@
+package seedu.duke.data;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.commands.Command;
+import seedu.duke.commands.CommandResult;
+import seedu.duke.commands.EditCommand;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EditCommandTest {
+
+    ShoppingList items = new ShoppingList();
+    Command command = new Command();
```

Comment:
I don't think you need to initialise the command here since in each individual test you will do something like 
Command command = new EditCommand(1, "apples", null);

Reply:
okie thank you for that suggestion!